modules/services/tox-node.nix

```diff
@@ -126,7 +126,7 @@ in {
   config = lib.mkIf cfg.enable {
     users.users."tox-node" = {
       description = "Tox node daemon user";
-      uid = config.ids.uids.tox-node;
+      uid = config.ids.uids.tox-bootstrapd;
```

Comment:
why?

Reply:
`config.ids.uids` are predefined in nixpkgs. Right now it's handy to just use `tox-bootstrapd` uid.

Reply:
Why? What for?

Reply:
To avoid `error: attribute 'tox-node' missing, at /nix/store/qc4iwishc810a38nincfm8lf6nlv9im4-source/modules/services/tox-node.nix:129:13`